### PR TITLE
Bare Metal Forms and Helpers Feature

### PR DESCRIPTION
**Pull request by dasileker**
Forms

### build with: 

- Rails
- Ruby
- Rubocop
- Linters

1. Created create, new, edit and update methods

2. created new and edit form

3. Used form_for

### CODE REVIEW

File: app/controllers/users_controller.rb
Comment (teekaytech) [inline app/controllers/users_controller.rb]: You can just pass in the user_params method you created below.
i.e.
`@user = User.new(user_params)`

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've made laudable progress in this project.
However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] Files not related to the project should be added to the .gitignore file.
- You added your `node_modules` to the repo. This is not a good practice and should be absolutely avoided. Kindly delete this file and add it to your .gitignore file, if you've not done so.

- [x] You mentioned that Ruby v2.6.3 is used to build the project in your README file whereas, 2.7.2 was used in your gemfile. Kindly update your readme.

- [x]  Use user_params to initialize user, i.e @user = User.new(user_params) 

- [x] I could not successfully create a new user from your form. Kindly inspect the create action in your users_controller to fix this issue.

- [x] Adjust the labels of your form elements. They should be to the left of the text boxes.

- [x] You are to comment out the `form` tag and not delete it completely ([Screenshot1](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form1.png) & [Screenshot2](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form2.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

- [x] You should also comment out the `form_tag` code and not delete it totally. ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_tag2.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by voscarmv — APPROVED** [review]
# Project status: Approved ✨ 😎👍 💯 ✔️

- Good work applying all the necessary changes.

### You may merge this branch and move forward ⛙🛣

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**
